README.md

```diff
@@ -219,73 +128,16 @@ bug and test the latest stable version.
 *Note: Your system libraries or preferences are not altered. In case you'd like to remove Hotspot again,
 simply delete the downloaded file. Learn more about AppImage [here](http://appimage.org/).*
 
-#### Debugging the AppImage
-
-When the AppImage crashes or is excessively slow, please provide a usable backtrace or profile run.
-To do that, you will need to download the debuginfo artifact that matches the AppImage you downloaded.
-Next, you will need to download the docker image from `ghcr.io/kdab/kdesrc-build:latest`. For that you
-need to authenticate yourself to GitHub. If you have a `ghcr.io` authentication token and use `pass`,
-you can do that with
+To find out how to debug the Appimage, see [HACKING#debugging-the-appimage].
```

Comment:
this should be `[HACKING.md#debugging-the-appimage]`, or?

Reply:
Not sure, it worked fine that way, but I'll recheck